CONTRIBUTING.rst

```diff
@@ -1153,6 +1153,32 @@ Here are a few rules that are important to keep in mind when you enter our commu
  * It’s OK to express your own emotions while communicating - it helps other people to understand you
  * Be considerate for feelings of others. Tell about how you feel not what you think of others
 
+Committer Responsibilities
+==========================
+
+Committers are more than contributors. While it's important for committers to maintain standing by
+committing code, their key role is to build and foster a healthy and active community.
+This means that committers should:
+
+* Review PRs in a timely and reliable fashion
+* They should also help to actively whittle down the PR backlog
+* Answer questions (i.e. on the dev list, in PRs, in Github Issues, slack, etc...)
+* Take on core changes/bugs/feature requests
+* Some changes are important enough that a committer needs to ensure it gets done. This is especially
+  the case if no one from the community is taking it on.
+* Improve processes and tooling
+* Refactoring code
+
+Commit Policy
+=============
+
+The following commit policy passed by a vote 8(binding FOR) to 0 against on May 27, 2016 on the dev list:
+
+* Commits need a +1 vote from a committer who is not the author
+* Furthermore, do not +1 or merge a PR that regresses linting or does not pass CI tests.
```

Comment:
We often do this when it's only a single job today fails with an obvious transient error, so we should change this process.

Also a +1 is separate from a merge - +1 days I am happy with the approach, but says nothing about test results. Again we all do this all the time.

Reply:
Agree. Just copied it from 2016. I am happy to change it to "Raality". I assume we do not have to vote on it since we are all doing it.